includes/class-llms-block-templates.php

```diff
@@ -0,0 +1,564 @@
+<?php
+/**
+ * Handles the block templates
```

Comment:
```suggestion
 * LLMS_Block_Templates class file.
```

Over time I've been switching the way the header matter is defined because the developer docs *do not parse* the file header. Adding a better description to the class header is then output on the parsed site at developer.lifterlms.com